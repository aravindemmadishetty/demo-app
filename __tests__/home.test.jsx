import { render, screen } from "@testing-library/react";
import Home from "../pages/index.js";
import "@testing-library/jest-dom";

test("rendering home page", () => {
  render(<Home />);
  expect(screen.getByText(/hello world/i)).toBeInTheDocument();
});

it("renders homepage unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
