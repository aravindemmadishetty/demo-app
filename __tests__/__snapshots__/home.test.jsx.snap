// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-between p-24 className"
  >
    <div>
      Hello World!
    </div>
  </main>
</div>
`;
